cranelift/codegen/src/context.rs

```diff
@@ -51,9 +50,9 @@ pub struct Context {
     pub loop_analysis: LoopAnalysis,
 
     /// Result of MachBackend compilation, if computed.
-    pub mach_compile_result: Option<MachCompileResult>,
+    compiled_code: Option<CompiledCode>,
```

Comment:
Why is this made private?

Reply:
Is there such an instance of an out-of-tree backend? Thought a getter would be sufficient, as suggested previously.

Reply:
Not that I know of. It might exist in the future. Also there may be benefits to moving all existing backends to a separate crate to make compilation of cranelift itself faster by allowing more parallism.

Reply:
Then it makes sense to add some kind of setter whenever it is required. (Good point about moving Cranelift backends to other crates, though; that would certainly help compiling it faster.)